bin/flutterflow_cli.dart

```diff
@@ -98,12 +111,18 @@ ArgResults _parseArgs(List<String> args) {
       defaultsTo: false,
     );
 
+  final firebaseDeployCommandParser = ArgParser()
```

Comment:
Do we need to support deploying from a branch?

Reply:
Don't think this matters for now, since firebase related stuff is all locked down for editing outside of the main branch.